### DISCUSSION TITLE
K8s 1.22 - Formalize docs release strategy with docs localization owners

### DISCUSSION THREAD

**Discussion by PI-Victor** [kubernetes/sig-release#1554] [General]
Hello 👋, I'm Victor Palade, the SIG Release Docs lead for the 1.22 release. I'm opening this discussion to coordinate the docs release (scheduled tentatively for Wed, August 4, [please see the release timeline](https://github.com/kubernetes/sig-release/tree/master/releases/release-1.22#tldr))

 I look forward to working with everyone! Let me know if you have any questions! Thank you! 🙏

cc: @kubernetes/sig-docs-fr-owners @kubernetes/sig-docs-de-owners @kubernetes/sig-docs-hi-owners @kubernetes/sig-docs-id-owners @kubernetes/sig-docs-ja-owners @kubernetes/sig-docs-ru-owners @kubernetes/sig-docs-pl-owners @kubernetes/sig-docs-ko-owners @kubernetes/sig-docs-pt-owners @kubernetes/sig-docs-es-owners @kubernetes/sig-docs-zh-owners @kubernetes/sig-docs-vi-owners @kubernetes/sig-docs-uk-owners @kubernetes/sig-docs-it-owners

**Comment by seokho-son** [comment]
Hello Victor. I am Seokho. I am leading SIG-Docs Korean Team :) Thank you for sharing~!

**Comment by mfilocha** [comment]
I'm here for Polish localization.

**Comment by raelga** [comment]
Hopping in for the spanish team.

**Comment by fabriziopandini** [comment]
Checking for the Italy team

**Comment by girikuncoro** [comment]
Checking in for the Indonesian team

**Comment by inductor** [comment]
Thanks from the Japanese team

**Comment by PI-Victor** [comment]
Hello localization team leads! I don't think any action is required from you, but I wanted to let you know that we are on track for the release on August 4, 2021 and all Kubernetes website branches are up to date (main, dev-1.22). Let me know if I can help with anything! Thanks!

**Reply by seokho-son** [reply]
(Korean l10n team) Thank you for sharing ! 
Do you have a plan to freeze k/website repository for merging dev-1.22 into main ?

**Reply by PI-Victor** [reply]
yes, please see the announcement in this thread

**Comment by PI-Victor** [comment]
❄️ Hello everyone, in order to prepare and to release v1.22, the `k/website` is now frozen till release day - 4th of August. During this time, No PRs will be merged into k/website!  ❄️

**Comment by PI-Victor** [comment]
Hello folks, the 1.22 release is live and k/website is now thawed, please feel free to merge at will
